README.md

```diff
@@ -1,2 +1,213 @@
-This Application is Build with React that is Web Application Containing Calculator that performs Math Operations.
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <img src="wahab_icon.jpeg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Math Magicians React Project Module 3</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Microverse Math Magicians React Project Module 3 ](#-microverse-math-magicians-react-project-module-3-)
+  - [🛠 Built With ](#-built-with-)
+    - [1- Html](#1--html)
+    - [2- CSS](#2--css)
+    - [3- Javascript](#3--javascript)
+    - [4- NPM](#4--npm)
+    - [5- React](#5--react)
+    - [6- Redux](#6--redux)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [1- Web Browser](#1--web-browser)
+    - [2- Code Editor](#2--code-editor)
+    - [3- Git](#3--git)
+    - [4- Node.js](#4--nodejs)
+    - [5- NPM](#5--npm)
+    - [6- React](#6--react)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Microverse Math Magicians React Project Module 3 <a name="about-project"></a>
+
+This is My Math Magicians Project of Module 3 in Microverse which is A Dynamic and Responsive Web Application that allows the user to use calculator to perform simple Calculations. it contains 3 components home, calculator, quote. it is built with React and Redux.   
+
+## 🛠 Built With <a name="built-with"></a>
+
+### 1- Html
+### 2- CSS
+### 3- Javascript
+### 4- NPM
+### 5- React
+### 6- Redux
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a>Html</a></li>
+    <li><a>Css</a></li>
+    <li><a>Javascript</a></li>
+    <li><a>NPM</a></li>
+    <li><a>React</a></li>
+    <li><a>Redux</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Calculator**
+- **Friendly UX**
+- **Easy to use**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### 1- Web Browser
+### 2- Code Editor
+### 3- Git
+### 4- Node.js
+### 5- NPM
+### 6- React
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run This Command:
+
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/Abdulwhbhussain/todo-list-webpack5.git
+```
```

Comment:
- [ ] [OPTIONAL] Please consider updating your README with the correct details to set up your project on a local machine 👍

Reply:
Sure!